src/AutoRest.CSharp/Common/AutoRest/Plugins/NewProjectScaffolding.cs

```diff
@@ -138,6 +138,28 @@ private string GetChangeLog()
 
         private string GetReadme()
         {
+            const string multipleApiVersionContent = @"
+
+### Service API versions
+
+The client library targets the latest service API version by default. A client instance accepts an optional service API version parameter to specify which API version service to communicate.
```

Comment:
```suggestion
The client library targets the latest service API version by default. A client instance accepts an optional service API version parameter from its options to specify which API version service to communicate.
```

The current wording makes it sound like this is a direct parameter to a client rather than needing to be set on the associated options type.